src/Gui.java

```diff
@@ -1,18 +1,54 @@
 import javax.swing.*;
 import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
 
-public class Gui {
+public class Gui implements ActionListener {
+    public void actionPerformed (ActionEvent e) {
+    }
     public static void main(String[] args) {
-        JFrame j = new JFrame();
-        j.setSize(500, 500);
-        j.getContentPane().setBackground(Color.white);
-        j.setVisible(true);
+        JFrame f = new JFrame();
+        f.setSize(500, 500);
+        f.getContentPane();
+        f.setBackground(Color.white);
+        f.setVisible(true);
 
-        JLabel Quizard = new JLabel("Quizard");
-        Quizard.setBounds(100, 100, 200, 200);
+        JPanel p = new JPanel();
```

Comment:
Let's create separate files for the different panels

While having it here is fine for now, as we work to add more and more panels, it's going to become a mess

Reply:
Also be more specific with your naming pls 

I,e since this is the panel for the start menu name smt it smt like startMenuJPanel